templates/admin/reporting/nav-filters.php

```diff
@@ -5,7 +5,7 @@
  * @package LifterLMS/Templates/Admin
  *
  * @since Unknown
- * @version 3.29.0
+ * @version [version]
```

Comment:
can you add a small changelog here?
e.g.
```suggestion
 * @since [version] What I did.
 * @version [version]
```

Reply:
done